client/service/apis.go

```diff
@@ -1,130 +1,327 @@
 package service
 
-var scimHeaders = map[string]string{
-	"Content-Type": "application/scim+json",
+import (
+	"crypto/tls"
+	"encoding/base64"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/mitchellh/go-homedir"
+	"gopkg.in/ini.v1"
+)
+
+// DatabricksClient is the client struct that contains clients for all the services available on Databricks
+type DatabricksClient struct {
+	Host               string
+	Token              string
+	Username           string
+	Password           string
+	Profile            string
+	ConfigFile         string
+	AzureAuth          AzureAuth
```

Comment:
should this be a pointer or the actual value?

Reply:
for this case no, because we have azure auth visitors